### PR TITLE
Login logic

### PR DESCRIPTION
**Pull request by HUSKUT**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+API_URL: http://localhost:8000/
```
Comment (SophiaH67) [inline .env.example:1]: Werkt dit nou wel of niet?

File: App.tsx [hunk 2]
```diff
@@ -1,87 +1,101 @@
...
 
-let isLoggedIn = false; // For testing, will require further programming
+// Still TBD on how we do this
+// and if its needed.
+const validateToken = () => {
+  getToken().then(async token => {
+    if (token == (await backendFetch('GET', 'api-auth-token'))) {
+    }
+  });
+};
```
Comment (SophiaH67) [inline App.tsx]: What does this function actually do?

File: App.tsx [hunk 3]
```diff
@@ -1,87 +1,101 @@
...
+
+let isLoggedIn = false;
+getToken().then(token => {
+  console.log('hi2');
+  if (token) {
+    isLoggedIn = true;
+  }
+});
```
Comment (SophiaH67) [inline App.tsx]: So, isLoggedIn will always be false during render? I'm a bit confused as to what this is supposed to do

Also left a console.log in there

File: .env.example (hunk 1 above)
Comment (HUSKUT) [reply .env.example:1]: nee op dit moment *nog* niet.

**Comment by HUSKUT** [comment]
ffs I somehow managed to fuck up in replying on @marnixah 's comments lol

**Comment by HUSKUT** [comment]
Made it a draft since I'm not entirely happy on the method I used

File: App.tsx (hunk 2 above)
Comment (HUSKUT) [reply App.tsx]: its a function meant to validate the token.

Yet I was unsure if we wanted that when I wrote it. I kept it in there for discussion sake.

File: App.tsx (hunk 3 above)
Comment (HUSKUT) [reply App.tsx]: If token exists
isloggedin value gets overridden.

ezpz

Comment (SophiaH67) [reply App.tsx]: Correct, however, this will not cause a re-render, since it's not a variable initialized with useState()

**Review by SophiaH67 — APPROVED** [review]

File: App.tsx (hunk 3 above)
Comment (HUSKUT) [reply App.tsx]: fixed in [7d357ce](https://github.com/TeamBAMI/frontend/pull/12/commits/7d357ce58dbb295e96e7563a0e0f06b6a7d72658)

**Review by ACodingPhoenix — APPROVED** [review]
yes.

**Pull request merged**